jetbrains-core/src/software/aws/toolkits/jetbrains/settings/AwsApplicationSettings.kt

```diff
@@ -0,0 +1,35 @@
+// Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.settings
+
+import com.intellij.openapi.components.PersistentStateComponent
+import com.intellij.openapi.components.ServiceManager
+
+class AwsApplicationSettings : PersistentStateComponent<AwsApplicationConfiguration> {
```

Comment:
I think we probably might want to separate this into SAM settings - breaking it up down the line won't be backwards compatible and as the toolkit grows we may want to break this up.

Reply:
Ya, easy enough. Will just rename this to SamSettings and also add storage annotations for aws.xml

Reply:
SamSettings now